docs/nerf_dataset_tips.md

```diff
@@ -53,11 +53,13 @@ See [nerf_loader.cu](src/nerf_loader.cu) for implementation details and addition
 
 ## Preparing new NeRF datasets
 
-To train on self-captured data, one has to process the data into an existing format supported by Instant-NGP. We provide scripts to support two complementary approaches:
+To train on self-captured data, one has to process the data into an existing format supported by Instant-NGP. We provide scripts to support three approaches:
 - [COLMAP](#COLMAP) to create a dataset from a set of photos or a video you took
 
 - [Record3D](#Record3D) to create a dataset with an iPhone 12 Pro or newer (based on ARKit)
 
+- [NeRFCapture](https://github.com/jc211/NeRFCapture) to create a dataset or stream posed images directly to InsantNGP with an iOS device.
```

Comment:
Should link to the #NeRFCapture section (like the other bullets link to their sections). Of course do link to the NeRFCapture app in the respective section's installation instructions.

Also, for consistency, __instant-ngp__ should be spelled lowercase and boldface. (Looking at the file, I realize that I have been sloppy with this in the past, spotting tome occurrences of "Instant-NGP")